README.md

```diff
@@ -136,17 +136,4 @@ docker run gcr.io/gcp-variant-transforms/gcp-variant-transforms:latest \
 
 ## Development
 
-### Testing
-
-To run all tests:
-
-```bash
-python setup.py test
-```
-
-To run a specific test:
-
-```bash
-python setup.py test -s <module>.<test class>.<test method>
-```
-
+Please see [Development Guide](https://github.com/googlegenomics/gcp-variant-transforms/blob/master/docs/DEVELOPMENT_GUIDE.md).
```

Comment:
nit: please put [Development Guide](...) in a new line all by itself as it's a long line (I recall this from some styleguide for .md files).

Reply:
Done.